packages/upload-api/src/types.ts

```diff
@@ -84,7 +84,13 @@ export interface CarStoreBucket {
   createUploadUrl: (
     link: UnknownLink,
     size: number
-  ) => Promise<{ url: URL; headers: { 'x-amz-checksum-sha256': string } }>
+  ) => Promise<{
+    url: URL
+    headers: {
+      'x-amz-checksum-sha256': string
+      'content-length': string
```

Comment:
what motivated adding this?

Reply:
Because w3infra passes `content-length` as number which was not included here. This made type checker pass but test fail because it checked against string. If this type existed this would have been caught without back and forth.